draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -2297,16 +2517,18 @@ client port 2 on one muxponder can be connected only with client port
 In case of flexible configuration, since the two muxponders are under
 the control of the same O-PNC, the configuration of the multiplexing
 label, regardless of whether it is a standard or vendor-specific
-label, can be done by the O-PNC using mechanisms which are vendor-specific and outside the scope of this document. The MDSC can just
-request the O-PNC to setup a client connectivity service over a WDM
+label, can be done by the O-PNC using mechanisms which are
+vendor-specific and outside the scope of this document. The MDSC can just
+request the O-PNC to setup a client connectivity service over a DWDM
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
request the O-PNC to setup a client connectivity service over a WDM
```